polymorphia-frontend/interfaces/api/knowledge-base/index.ts

```diff
@@ -1,6 +1,6 @@
-import { BaseReward } from "../reward";
+import {BaseReward} from "../reward";
+import {KnowledgeBaseType} from "@/components/slider/types";
```

Comment:
```suggestion
import { BaseReward } from "../reward";
import { KnowledgeBaseType } from "@/components/slider/types";
```
nitpick: formatting, Prettier should catch this (`npm run prettier:check`, `npm run prettier:fix`)